source/contracts/trading/ClaimProceeds.sol

```diff
@@ -51,11 +56,19 @@ contract ClaimProceeds is CashAutoConverter, ReentrancyGuard, IClaimProceeds {
             if (_reporterShare > 0) {
                 require(_denominationToken.transferFrom(_market, _market.getUniverse().getNextReportingWindow(), _reporterShare));
             }
+            
```

Comment:
Nitpick: No indentation on blank lines.  I'm surprised your editor let you save the file like this...

Reply:
Hm I must not have the right setting in vscode. Will add and correct